packages/json-machete/src/healJSONSchema.ts

```diff
@@ -368,6 +368,11 @@ export async function healJSONSchema(
             logger.debug(`${path} has an enum but with a single value. Converting it to const.`);
             delete subSchema.enum;
           }
+          if (subSchema.enum && subSchema.enum.includes(null)) {
+            logger.debug(`${path} has "null" value. Converting it to nullable.`);
+            subSchema.enum = subSchema.enum.filter((e: unknown) => typeof e === 'string');
```

Comment:
I think enums can be any primitive(string, int32, int64 etc) per JSON Schema spec.
This might break enums that have numbers instead of strings.